### PR TITLE
Features: Improved graph physics, added description to graph page and peers are sorted by uptime

### PR DESCRIPTION
**Pull request by sourencho**

### CODE REVIEW

File: build/app.js [hunk 1]
```diff
@@ -58410,13 +58410,21 @@ app.controller('PeersCtrl', ['$scope', function ($scope) {
   $scope.status = "Loading Peers..."
 
   Peers.fetch().then(function(peers) {
```
Comment (stevenzeiler) [inline build/app.js:58412]: Why don't you put the sorting directly in Peers.fetch() so it can live in one place?

Comment (sourencho) [inline build/app.js:58412]: you mean as a separate function from `fetch`?

Comment (stevenzeiler) [inline build/app.js:58412]: I meant inside fetch() so results are always sorted. Either way is fine with me, and sorting in the controller definitely makes sense too.

**Comment by stevenzeiler** [comment]
:+1: LGTM

**Pull request merged**